en/components/grid_summaries.md

```diff
@@ -155,43 +136,46 @@ export class GridComponent implements OnInit {
 }
 ```
 
-It is good to keep in mind that in order to improve performance, **igx-grid** cache all summaries and recalculate them if you perform CRUD operations over your data. But if your data source is modified outside the **igx-grid**, you need to explicitly force the **igx-grid** to recalculate your summaries by invoking the method `clearSummaryCache()`.
+### Summaries with Group By
 
-```html
-<igx-grid #grid1 [data]="data" [autoGenerate]="false" height="800px" width="800px" (onColumnInit)="initColunm($event)" >
-    <igx-column field="ProductID" width="200px"  [sortable]="true">
-    </igx-column>
-    <igx-column field="ProductName" width="200px" [sortable]="true" [hasSummary]="true">
-    </igx-column>
-    <igx-column field="UnitsInStock" width="200px" [dataType]="'number'" [hasSummary]="true" [summaries]="mySummary" [sortable]="true">
-    </igx-column>
-    <igx-column field="ReorderLevel" width="200px" [editable]="true" [dataType]="'number'" [hasSummary]="true">
-    </igx-column>
-</igx-grid>
-<button (click)="updateData()">Update Data</button>
-```
+When you have grouped by columns, the grid allows you to change the summary position and calculation mode using the [`summaryCalculationMode`]({environment:angularApiUrl}/classes/igxgridcomponent.html#summaryCalculationMode) and [`summaryPosition`]({environment:angularApiUrl}/classes/igxgridcomponent.html#summaryPosition) properties.
 
-```typescript
-...
-export class GridComponent implements OnInit {
+The available values of the [`summaryCalculationMode`]({environment:angularApiUrl}/classes/igxgridcomponent.html#summaryCalculationMode) property are:
```

Comment:
**#summaryCalculationMode** should be lower case to navigate correctly

Reply:
Done.